packages/components/src/organisms/ImageGallery/ImageGallery.tsx

```diff
@@ -51,6 +55,7 @@ const ImageGallery = forwardRef<HTMLDivElement, ImageGalleryProps>(
       children,
       ImageComponent,
       selectedImageIdx,
+      imagePosition = 'center',
```

Comment:
When adding the override option, the chain condition will be like:

`cmsValue ?? override ?? 'center'`

This makes so this default should probably be set on the component that uses it, not here.